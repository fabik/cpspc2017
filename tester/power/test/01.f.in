81
9 5
2 8
1 9
6 8
4 5
6 6
3 3
8 7
7 1
5 2
1 3
9 1
7 3
3 7
9 6
3 5
5 3
2 5
5 6
2 7
7 9
1 8
7 2
9 9
8 6
9 4
1 4
6 4
7 4
6 3
2 1
4 6
3 9
6 1
7 6
8 3
5 7
6 2
8 8
2 6
5 4
9 3
4 1
3 2
5 8
8 2
9 8
2 2
3 8
4 9
1 1
1 5
4 8
2 9
8 5
7 5
1 7
9 2
8 1
8 4
7 7
4 4
8 9
5 5
4 3
9 7
4 7
4 2
2 3
7 8
3 1
6 9
3 4
5 1
5 9
6 5
6 7
1 6
2 4
1 2
3 6
