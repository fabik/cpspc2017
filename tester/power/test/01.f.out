2
41
1 2 3 4 6 7 9 11 12 13 14 16 17 21 24 25 28 32 33 37 38 39 40 42 46 48 51 52 53 56 57 60 61 62 64 66 68 71 74 75 79
40
5 8 10 15 18 19 20 22 23 26 27 29 30 31 34 35 36 41 43 44 45 47 49 50 54 55 58 59 63 65 67 69 70 72 73 76 77 78 80 81
